core/commands/storage/announce.go

```diff
@@ -70,13 +88,48 @@ $ btfs storage announce --host-storage-price=1000000`,
 			}
 		}
 
+		rm, rmFound := req.Options[repairHostEnabledOptionName].(bool)
+		if rmFound {
+			if rm != cfg.Experimental.HostRepairEnabled {
+				cfg.Experimental.HostRepairEnabled = rm
+				err = n.Repo.SetConfig(cfg)
+				if err != nil {
+					return err
+				}
+			}
+			if !rm {
+				return nil
+			}
```

Comment:
This does not apply here. Even if its turned off, the other parameters can still be set.